### ISSUE TITLE
GTK-Message on cinnamon-settings-users.py

### ISSUE THREAD

**Issue by abdouskamel** [linuxmint/Cinnamon#6976]
```
 * Cinnamon version - cinnamon 3.2.7
 * Distribution - Debian 9.2
 * 64 bit
 ```

**Issue**

It's an issue about **cinnamon-settings-users.py** located in _/usr/share/cinnamon/cinnamon-settings-users_. When the program is opening a dialog, the following message is printed is the terminal : _Gtk-Message: GtkDialog mapped without a transient parent. This is discouraged._

**Steps to reproduce**

1. Open **cinnamon-settings-users.py** located in _/usr/share/cinnamon/cinnamon-settings-users_ with a terminal.
2. Open a dialog by adding a new user or a new group.

**Expected behaviour**

The message : _Gtk-Message: GtkDialog mapped without a transient parent. This is discouraged._ on the terminal.

**Comment by abdouskamel** [comment]
Just created a pull request : #6977.

**Comment by leigh123linux** [comment]
cinnamon 3.2.7 is EOL, if you want it fixed report it to debian bug tracker!

**Comment by abdouskamel** [comment]
I get the same result by using the python script available in this repository : [here it is](https://github.com/linuxmint/Cinnamon/blob/master/files/usr/share/cinnamon/cinnamon-settings-users/cinnamon-settings-users.py). So i guess we still need to fix it in the latest version of cinnamon.

**Comment by abdouskamel** [comment]
It's just about creating a dialog without specifying its parent. It can cause semantic problems in the visual tree of the main window. It was easy to fix, you can see pull request : #6977.
